elemental-iso-build

```diff
@@ -25,7 +27,7 @@ build() {
 
   rm -Rf "$TEMPDIR"
   ISONAME="elemental-$(date "+%FT%TZ")"
-  docker run --rm -v "$PWD":/mnt elemental/iso:latest build-iso -o /mnt --squash-no-compression -n "$ISONAME" --overlay-iso overlay dir:rootfs
+  docker run --privileged --rm -v "$PWD":/mnt elemental/iso:latest build-iso -o /mnt --squash-no-compression -n "$ISONAME" --bootloader-in-rootfs --overlay-iso /iso/overlay dir:rootfs
```

Comment:
This --privileged is needed to build legacy firmware ISOs.... as we need to call `grub2-mkimage` in a chroot for that. I am wondering to completely remove BIOS support when using the `--bootloader-in-rootfs` flag, since the previous strategy of pulling artifacts from somewhere else is still possible it will not be a feature loss in any case.

Reply:
that is kind of bad, should not need --privileged to build an iso :/

Is there any way we can circumvent that?

Reply:
Yes, I am on it. Building for EFI or BIOS firmware images, but not for both... Gonna fix it a follow up PR though. Only BIOS requires privileges.

Reply:
we should probably look into dropping bios shortly blargh